simulator/modules/fetch/bpu/t/unit_test.cpp

```diff
@@ -52,6 +64,10 @@ TEST_CASE( "One bit predictor")
     bp->update( BPInterface( PC, true, target));
     CHECK( bp->is_taken(PC) );
     CHECK( bp->get_target(PC) == target);
+
+    target = 16;
+    bp->update( BPInterface( PC, true, target));
```

Comment:
There should be a dedicated test case for that behavior. Good practice is to have only one 'acting-assertion' section in each test case.

Reply:
Ok, I will correct it.